Source/Core/VideoCommon/PerformanceMetrics.cpp

```diff
@@ -161,27 +160,29 @@ void PerformanceMetrics::DrawImGuiStats(const float backbuffer_scale)
       ImGui::SetWindowPos(ImVec2(clamped_window_x, clamped_window_y), ImGuiCond_Always);
   };
 
-  const float graph_width = 50.f * backbuffer_scale + 3.f * window_width + 2.f * window_padding;
-  const float graph_height =
-      std::min(200.f * backbuffer_scale, display_size.y - 85.f * backbuffer_scale);
+  const float graph_width = display_size.x / 4.0;
+  const float graph_height = display_size.y / 4.0;
 
   const bool stack_vertically = !g_ActiveConfig.bShowGraphs;
 
   ImGui::PushStyleVar(ImGuiStyleVar_WindowBorderSize, 0.f);
   ImGui::PushStyleVar(ImGuiStyleVar_WindowRounding, 14.f * backbuffer_scale);
   if (g_ActiveConfig.bShowGraphs)
   {
+    // Force smaller font
+    ImGui::PushFont(NULL, 13.0f);
```

Comment:
The default value should probably be hardcoded somewhere and that should be used here